service/src/main/java/com/fecklessweasel/service/objectmodel/UserSession.java

```diff
@@ -72,7 +72,7 @@ public static UserSession resume(Connection connection,
             throw new ServiceException(ServiceStatus.ACCESS_DENIED);
```

Comment:
What if the server is just down? Maybe ACCESS_DENIED isn't the correct error message.